tests/Mapper/ProgrammesDbToDomain/CollapsedBroadcastMapperTest.php

```diff
@@ -53,8 +53,8 @@ public function testGetDomainModel()
 
         $dbEntityArray = [
             'id'               => 1,
-            'startAt'          => new DateTime('2015-01-03T00:00:00'),
-            'endAt'            => new DateTime('2015-01-03T01:00:00'),
+            'startAt'          => new DateTimeImmutable('2015-01-03T00:00:00', new DateTimeZone('UTC')),
```

Comment:
I wondered what the Z was for...  
Is Zulu forever UTC? It has no daylight savings?